src/SFA.DAS.ApprenticeCommitments/Data/Models/ApprenticeCommitmentsDbContext.cs

```diff
@@ -18,6 +18,10 @@ public ApprenticeCommitmentsDbContext(DbContextOptions<ApprenticeCommitmentsDbCo
         public virtual DbSet<Apprentice> Apprentices { get; set; }
         public virtual DbSet<Apprenticeship> Apprenticeships { get; set; }
 
+        DbSet<Registration> IEntityContext<Registration>.Entities => Registrations;
+        DbSet<Apprentice> IEntityContext<Apprentice>.Entities => Apprentices;
+        DbSet<Apprenticeship> IEntityContext<Apprenticeship>.Entities => Apprenticeships;
+
```

Comment:
These statements intrigue me on how the interfaces resolve it... but I can guess. Again it's an interesting example of new language features.

Reply:
It's via C#'s [Explicit Interface Implementation](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation) - which as far as I'm aware has always been in the language.  It's not something I have need to use very often though.